### PR TITLE
fix: update error codes in batch to match unbatched format

### PR DESCRIPTION
**Pull request by alexromano**
corresponding doc changes for https://github.com/Finch-API/api-server/pull/3630

### CODE REVIEW

**Comment by alexromano** [comment]
Screen Shot 2022-11-28 at 2 20 59 PM
Screen Shot 2022-11-28 at 2 21 24 PM

File: docs/Development-Guides/Errors/Error-Handling.md [hunk 1]
```diff
@@ -48,12 +48,16 @@ Response Body:
...
-        "error_name": "server_error",
-        // corresponds to the `message` parameter of the error schema in the Error Types guide
-        "error_message": "Internal Server Error"
+        "name": "not_found_error",
+        "code": 404,
+        "finch_code": "individual_not_found",
+        "message": "The individual with id 'fbeabe51-e6d2-45aa-a460-4c8482528f41' could not be found",
+        // deprecated field. corresponds to the `name` parameter of the error schema in the Error Types guide
+        "error_name": "not_found_error",
+        // deprecated field. corresponds to the `message` parameter of the error schema in the Error Types guide
```
Comment (aren55555) [inline docs/Development-Guides/Errors/Error-Handling.md:59]: Should we indicate a date when we anticipate we will remove this by?

**Review by aren55555 — APPROVED** [review]

File: docs/Development-Guides/Errors/Error-Handling.md (hunk 1 above)
Comment (alexromano) [reply docs/Development-Guides/Errors/Error-Handling.md:59]: yeah. what do you think is reasonable? 6 months?
we'll have to notify developers about this as well.

Comment (aren55555) [reply docs/Development-Guides/Errors/Error-Handling.md:59]: yeah 6mo sounds reasonable; June/July 2023?

**Pull request merged**